homeassistant/components/gpstracker/config_flow.py

```diff
@@ -0,0 +1,135 @@
+"""Config flow for GPS Tracker integration."""
+from __future__ import annotations
+
+import logging
+from typing import Any
+
+import aiohttp
+import gps_tracker
+import voluptuous as vol
+
+from homeassistant.config_entries import ConfigEntry, ConfigFlow
+from homeassistant.const import CONF_PASSWORD, CONF_USERNAME
+from homeassistant.core import HomeAssistant
+from homeassistant.data_entry_flow import FlowResult
+from homeassistant.exceptions import HomeAssistantError
+
+from .const import DOMAIN
+from .helpers import get_gpstracker_client
+
+_LOGGER = logging.getLogger(__name__)
```

Comment:
Import folleg from const.py

Reply:
Done in 24824a7206c5ad4c9a01d44d198eac07170d8184